### DISCUSSION TITLE
Welcome to Account-API Discussions!

### DISCUSSION THREAD

**Discussion by Arturr-H** [Arturr-H/Account-API#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LittleMe007** [comment]
I was hoping someone can point me in the right direction. I'm working on creating a web-based application to upload .csv, XML (part purchase orders, and part invoices) to and post the data into our backend system with an API they are providing us. What kind of development on the API and server must take place? Anything helps thanks!
